packages/@lwc/shared/src/language.ts

```diff
@@ -75,6 +89,7 @@ export {
     ArraySort,
     ArraySplice,
     ArrayUnshift,
+    arrayEvery,
```

Comment:
Let's use `ArrayEvery` for consistent capitalization

Reply:
I believe that would be more confusing. `arrayEvery !== Array.prototype.every` whereas `ArraySort === Array.prototype.sort`. Exposing `Array.prototype.every` as `ArrayEvery` causes a TypeScript error _in the generated `.d.ts`_, which can't be `@ts-ignore`d.

Since `arrayEvery` is not the same kind of thing as the other `CapitalizedExports`, it seems like capitalizing in that same way would be misleading to future users of `ArrayEvery`.

Reply:
I see, okay. Thanks for the explanation!